src/components/nav-sidebar/NavSidebar.scss

```diff
@@ -58,7 +59,7 @@
     }
 
     &.is-collapsed {
-        > ul,
+        :not(.is-animation-enabled) > ul,
```

Comment:
anyway to simplify by not using not?

Reply:
This is the most simple solution I could think of. The problem is that the library need to control the display of the element in order to properly animate. I also thought about overwriting the style, but thought this was cleaner.

Reply:
we have something very similar in the count badge component, for comparison